Tutorial/ReportPageTooltips.md

```diff
@@ -0,0 +1,41 @@
+# Adding Report Page Tooltips support to Bar Chart
+Report page tooltips support can be done by updating your visual capabilities.
+See [commit](https://github.com/Microsoft/PowerBI-visuals-sampleBarChart/commit/3c6e8186436b63bf0cf97d2cdd5dde8aa8d08709) for what was added at this step.
```

Comment:
Prior for checking this in, the link to sampleBarChart update **commit** guid will be corrected to the right one(once it will get into master)